embassy-stm32/src/rcc/g4.rs

```diff
@@ -262,20 +291,22 @@ pub(crate) unsafe fn init(config: Config) {
         }
     };
 
-    // Setup the 48 MHz clock if needed
-    if let Some(clock_48mhz_src) = config.clock_48mhz_src {
-        let source = match clock_48mhz_src {
-            Clock48MhzSrc::PllQ => {
+    // Configure the 48MHz clock source for USB and RNG peripherals.
+    {
```

Comment:
these `{}` are not needed.

Reply:
was intended to prevent `source` leaking into the parent namespace, but I’ll shift it around to not require binding to a variable at all